cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake

```diff
@@ -105,8 +105,11 @@ endif()
 
 if(NOT CUDA_VERSION VERSION_LESS "11.8")
   list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Ada")
+  list(APPEND CUDA_KNOWN_GPU_ARCHITECTURES "Hopper")
```

Comment:
We should add bin/ptx for Ada and Hopper in 
https://github.com/pytorch/pytorch/blob/95fc0bcaaddc2d24e8759f24dbefa789d04e9e42/cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake#L248-L250